src/apps/opening-hours-editor/helper.ts

```diff
@@ -172,3 +172,10 @@ export const isOpeningHourWeeklyRepetition = (
     DplOpeningHoursListGET200ItemRepetitionType.weekly
   );
 };
+
+// This used for cypress testing in in cms where we test the complete flow of
+// opening hours between the editor (Admin) and the calendar (Frontend)
+export const getInitialDateFromUrl = (): string | null => {
+  const query = new URLSearchParams(window.location.search);
+  return query.get("initialDate");
+};
```

Comment:
Please consider your typing.

As I see it you have a function `getInitialDateFromUrl()` which returns a string. This causes changes all the way down call stack which has been designed to accomodate a Date.

As I see it a Date is the proper type to use for this. Instead of changing the type I suggest that you ensure that the return value from `getInitialDateFromUrl()` is in fact a Date.

If that is not the case you can return null (and log the problem to `console.debug()`) or throw an error.